client/js/app.jsx

```diff
@@ -29,7 +29,7 @@ class Root extends React.Component {
     const { store } = this.props;
     return (
       <Provider store={store}>
-        <div>
+        <div style={{"height": "100%"}}>
```

Comment:
You could maybe add a container file to the components or something. I guess it is not that big of a deal though